fastlane/Fastfile

```diff
@@ -0,0 +1,148 @@
+      timeout: 3600,
+      add_to_search_list: true
+    )
+  end
+
+  desc "Fetches provisioning profile and certificates from github repo"
+  lane :certificates do |options|
+    export_method = options[:export_method]
+    match(
+      git_branch: ENV["TEAM_SAGEBIO_NOT_FOR_PROFIT"],
+      type: "development",
+      app_identifier: ENV["APP_ID_RSDCATALOG"],
+      team_id: ENV["TEAM_SAGEBIO_NOT_FOR_PROFIT"],
+      keychain_name: ENV["MATCH_KEYCHAIN_NAME"],
+      keychain_password: ENV["MATCH_PASSWORD"],
+      readonly: true
+    )
+    match(
+      git_branch: ENV["TEAM_SAGEBIO_NOT_FOR_PROFIT"],
+      type: "appstore",
+      app_identifier: ENV["APP_ID_RSDCATALOG"],
+      team_id: ENV["TEAM_SAGEBIO_NOT_FOR_PROFIT"],
+      keychain_name: ENV["MATCH_KEYCHAIN_NAME"],
+      keychain_password: ENV["MATCH_PASSWORD"],
+      readonly: true
+    )
+    match(
+      git_branch: ENV["TEAM_SAGEBIO_NOT_FOR_PROFIT"],
+      type: "development",
+      app_identifier: ENV["APP_ID_RSDTEST"],
+      team_id: ENV["TEAM_SAGEBIO_NOT_FOR_PROFIT"],
+      keychain_name: ENV["MATCH_KEYCHAIN_NAME"],
+      keychain_password: ENV["MATCH_PASSWORD"],
+      readonly: true
+    )
+    match(
+      git_branch: ENV["TEAM_SAGEBIO_NOT_FOR_PROFIT"],
+      type: "appstore",
+      app_identifier: ENV["APP_ID_RSDTEST"],
+      team_id: ENV["TEAM_SAGEBIO_NOT_FOR_PROFIT"],
+      keychain_name: ENV["MATCH_KEYCHAIN_NAME"],
+      keychain_password: ENV["MATCH_PASSWORD"],
+      readonly: true
+    )
+  end
+
+  desc "Execute tests"
+  lane :test do |options|
+    workspace = options[:workspace]
+    scheme = options[:scheme]
+    scan(
+      workspace: "#{workspace}",
+      scheme: "#{scheme}",
+      clean: true
+    )
+  end
+
+  desc "Archive and export app"
+  lane :archive do |options|
+    scheme = options[:scheme]
+    export_method = options[:export_method]
+    gym(
+      scheme: "#{scheme}",
+      clean: true,
+      include_bitcode: true,
+      output_directory: "./build",
+      output_name: "#{scheme}.ipa",
+      export_method: "#{export_method}",
+      destination: "generic/platform=iOS",
+    )
+  end
+
+  desc "This lane is for CI bots to archive and export"
+  lane :ci_archive do |options|
+    scheme = options[:scheme]
+    export_method = options[:export_method]
+    keychains
+    certificates
+    archive(scheme: "#{scheme}",export_method: "#{export_method}")
+  end
+
+  desc "Submit a new Build to appstore"
+  lane :beta do |options|
+    scheme = options[:scheme]
+    export_method = options[:export_method]
+    ensure_git_status_clean
+    keychains
+    certificates
+    version_number = get_version_number(xcodeproj: "#{project}", target: "#{scheme}")
+    next_build_number = increment_build_number
+    archive(scheme: "#{scheme}", export_method: "#{export_method}")
+    commit_version_bump(xcodeproj: "#{scheme}.xcodeproj", message: "[skip ci] Version Bump")
+    add_git_tag(tag: "#{version_number}_#{next_build_number}")
```

Comment:
Tags should include which schema is being bumped.

Also, need a tag for bumping the ResearchStack2UI and ResearchStack2 projects as well.